backend/lib/backend-stack.ts

```
@@ -0,0 +1,112 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { HttpApi, HttpMethod } from 'aws-cdk-lib/aws-apigatewayv2';
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
+import { Runtime, Function } from 'aws-cdk-lib/aws-lambda';
```

Biome (2.1.2)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)